package.json

```diff
@@ -7,7 +7,7 @@
     "dist/*"
   ],
   "scripts": {
-    "test": "jest",
+    "test": "jest --detectOpenHandles",
```

Comment:
please revert that before merging to master

Reply:
why?

Reply:
https://jestjs.io/docs/en/cli

> --detectOpenHandles
Attempt to collect and print open handles preventing Jest from exiting cleanly. Use this in cases where you need to use --forceExit in order for Jest to exit to potentially track down the reason. This implies --runInBand, making tests run serially. Implemented using async_hooks. This option has a significant performance penalty and should only be used for debugging.

Specifically
> Use this in cases where you need to use --forceExit in order for Jest to exit to potentially track down the reason.

Is this your use case? You should run this locally if so, fix the issue and run normally afterwards.